modules/installation-aws-permissions.adoc

```diff
@@ -70,8 +73,8 @@ cluster, the IAM user requires the following permissions:
 [%collapsible]
 ====
 * `ec2:AllocateAddress`
-* `ec2:AssociateAddress`
 * `ec2:AssociateDhcpOptions`
+* `ec2:AssociateAddress`
```

Comment:
Hm why this change? It looks like all it is doing is making the list non-alphabetically ordered?

Reply:
Hi @patrickdillon . Correct. I just changed the order to match https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go . Mkes comparing the permissions.go fiel and .adoc list a bit quicker.